cvat/apps/events/event.py

```diff
@@ -47,25 +47,27 @@ def record_server_event(
     *,
     scope: str,
     request_id: Optional[str],
+    user_agent: Optional[str] = None,
     payload: Optional[dict] = None,
     on_commit: bool = False,
     **kwargs,
 ) -> None:
     payload = payload or {}
 
-    payload_with_request_id = {
+    payload_with_request_info = {
         **payload,
         "request": {
             **payload.get("request", {}),
             "id": request_id,
+            "user-agent": user_agent,
```

Comment:
```suggestion
            "user_agent": user_agent,
```

(I'm pretty sure we're using snake_case in these fields... see `working_time`.)